README-local.md

```diff
@@ -287,6 +291,123 @@ extension=imagick
 
 Reboot your computer. Upon rebooting, the application should load as expected
 
+#### Logging in
+
+To log in locally, use one of the following test accounts
+
+- admin@test.com
+- manager@test.com
+- applicant@test.com
+- hr_advisor@test.com
+
+The password for each account is 'password'
+
+---
+
+## macOS
+These instructions were written using macOS High Sierra (Version 10.13.6)
+
+## Install homebrew
+
+*Further instructions available [here](https://brew.sh/)*
+
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+
+## Install nginx
+
+```
+brew install nginx
+brew services start nginx
+sudo ufw allow 'Nginx HTTP'
+```
+
+## Install PostGres
+
+```
+brew services start postgresql
+```
+
+### Configure postgres
+
+```
+psql -d postgres ## login to postgres
+create database talentcloud; ### create database
+create user talentcloud with encrypted password 'talentcloud'; ### create user
+grant all privileges on database talentcloud to talentcloud; ### adjust access
+ALTER ROLE talentcloud SUPERUSER;
+```
+
+## Install PHP and required extensions
+
+*Laravel-specific server requirements available [here](https://laravel.com/docs/8.x/installation#server-requirements)*
+
+```
+brew install php@7.2 --without-apache --with-postgresql
+brew install pkg-config imagemagick
+pecl install imagick
+```
+
+## Install node
+
+```
+brew install node@12
+```
+
+## Install composer
+
+```
+sudo brew install composer
+```
+
+## Run Talent Cloud
+
+### Clone repo
+
+#### HTTPS
+
+```
+git clone https://github.com/GCTC-NTGC/TalentCloud.git
+```
+
+#### SSH
+
+```
+git clone git@github.com:GCTC-NTGC/TalentCloud.git
+```
+
+#### Setup TalentCloud directory to be hosted
+
+*For more details, please visit [here](https://laravel.com/docs/8.x/valet#securing-sites)*
+
+```
+cd TalentCloud
+valet secure talent
+valet link talent
+valet restart
```

Comment:
Curious why you're installing nginx and running valet? AFAIK valet should take care of an nginx installation and running it in the background when the Mac starts. Looks like you can also tell valet to use a specific version of PHP via Homebrew according to [the docs](https://laravel.com/docs/7.x/valet#installation): `valet use php@7.2`.

Not opposed to one method or another, but it might make sense to outline these as alternatives rather than part of the same setup process.

Reply:
Thanks for the feedback @Khristophor. I have updated the instructions for the macOS portion to remove the part to install nginx via homebrew, added `valet use php@7.2` into the process once valet has been installed, and cleaned up a few other sections to hopefully make the process more straightforward.